tools/harness-simulation/posix/launch_testbed.py

```diff
@@ -194,11 +228,25 @@ def exit_handler(signum, context):
 
         if data == b'BBR':
             logging.info('Received OpenThread simulation query, advertising')
-            advertise_ftd(s, src, ven='OpenThread_Sim', ver='4', add=addr, por=22, number=args.ot11_num)
+            advertise_devices(s,
+                              src,
+                              ven='OpenThread_Sim',
+                              ver='4',
```

Comment:
Does the version matter?

Reply:
No, it does not matter. Acutally both `ver` and `por` do not matter. I will remove them from the arguments.

Reply:
The value in `ven` field is the name of the class that Harness will instantiate during testing. Therefore it has to be `OpenThread_Sim` or `OpenThread_BR_Sim`.

`mod`, `ver` and `por` are basically useless fields but have to be kept.